test/conftest.py

```diff
@@ -42,6 +42,15 @@
 from pudl.workspace.datastore import Datastore
 from pudl.workspace.setup import PudlPaths
 
+pudl.logging_helpers.configure_root_logger(
+    dependency_loglevels={
+        "numba": logging.DEBUG,
```

Comment:
Do we really want DEBUG logs for numba? It looks like we had it set to WARNING before.